lib/Config/Hooks.php

```diff
@@ -159,6 +159,16 @@ public function setupSubscriptionEvents() {
           3
         );
       }
+      $this->wp->addAction(
+        'woocommerce_register_form',
+        [$this->subscriptionRegistration, 'extendForm']
```

Comment:
I think this should go through WooCommerce hooks class so that it catches any errors, shouldn't it?